editions/tw5.com/tiddlers/gettingstarted/GettingStarted.tid

```diff
@@ -4,9 +4,14 @@ tags: [[Working with TiddlyWiki]]
 title: GettingStarted
 type: text/vnd.tiddlywiki
 
-Click here to download an empty copy of TiddlyWiki: {{$:/editions/tw5.com/snippets/download-empty-button}}
+!!!1. Download an empty copy of TiddlyWiki
 
-The next step is to choose a method for saving changes. There's a wide variety of methods available, with different features and limitations. Click on the badge for a method to see more information about it. You can also click on one of the platform filters to restrict the listing to methods that work with that platform.
+<div style="display: flex; justify-content: center;">{{$:/editions/tw5.com/snippets/download-empty-button}}</div>
```

Comment:
OK, I read elsewhere that we categorically don't use inline styling ever in the docs. I'll change this (...but I'd still appreciate feedback if there's anything else that is not *better* than the current doc.)

Reply:
The comment you read was about inline style blocks (ie `<style>` elements in ordinary tiddlers), not about ordinary inline styling.